extensions/SharkPool/SoundCloud-API.js

```diff
@@ -0,0 +1,363 @@
+            blockType: Scratch.BlockType.REPORTER,
+            text: "get mp3 of track ID [ID]",
+            arguments: {
+              ID: { type: Scratch.ArgumentType.NUMBER, defaultValue: 241049935 }
+            }
+          },
+          {
+            opcode: "getTrackComment",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "get [NUM2] offset [NUM1] of [TYPE] comments from track ID [ID]",
+            arguments: {
+              TYPE: { type: Scratch.ArgumentType.STRING, menu: "COMMENT" },
+              ID: { type: Scratch.ArgumentType.NUMBER, defaultValue: 241049935 },
+              NUM1: { type: Scratch.ArgumentType.NUMBER, defaultValue: 0 },
+              NUM2: { type: Scratch.ArgumentType.NUMBER, defaultValue: 20 }
+            },
+          },
+          {
+            opcode: "searchTracks",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "search for tracks with query [QUERY]",
+            arguments: {
+              QUERY: { type: Scratch.ArgumentType.STRING, defaultValue: "Ancient Visions" }
+            }
+          },
+          { blockType: Scratch.BlockType.LABEL, text: "Artists" },
+          {
+            opcode: "getArtistAtt",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "get [THING] from artist ID [ID]",
+            arguments: {
+              THING: { type: Scratch.ArgumentType.STRING, menu: "ARTISTS" },
+              ID: { type: Scratch.ArgumentType.NUMBER, defaultValue: 127123168 }
+            }
+          },
+          {
+            opcode: "getFollowers",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "get [NUM2] offset [NUM1] of followers from artist ID [ID]",
+            arguments: {
+              ID: { type: Scratch.ArgumentType.NUMBER, defaultValue: 127123168 },
+              NUM1: { type: Scratch.ArgumentType.NUMBER, defaultValue: 0 },
+              NUM2: { type: Scratch.ArgumentType.NUMBER, defaultValue: 20 }
+            },
+          },
+          {
+            opcode: "getTracks",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "get [NUM2] offset [NUM1] of tracks from artist ID [ID]",
+            arguments: {
+              ID: { type: Scratch.ArgumentType.NUMBER, defaultValue: 127123168 },
+              NUM1: { type: Scratch.ArgumentType.NUMBER, defaultValue: 0 },
+              NUM2: { type: Scratch.ArgumentType.NUMBER, defaultValue: 20 }
+            },
+          },
+          {
+            opcode: "searchArtists",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "search for artists with query [QUERY]",
+            arguments: {
+              QUERY: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "Aliantos"
+              }
+            }
+          },
+        ],
+        menus: {
+          IDS: ["track", "artist"],
+          COMMENT: ["new", "old"],
+          TRACKS: {
+            acceptReporters: true,
```

Comment:
acceptReporters is kind of redundant here considering the values themselves are very ambiguous and with no rhyme or reason at all.